poulpe-service/src/main/java/org/jtalks/poulpe/service/transactional/TransactionalUserService.java

```diff
@@ -298,6 +300,17 @@ private PoulpeUser getPoulpeUser(String username) throws NotFoundException {
     @Override
     public void registration(PoulpeUser user) throws ValidationException {
         List<String> errors = new ArrayList<String>();
+
+        BindingResult validateErrors = new BeanPropertyBindingResult(user, "user");
```

Comment:
This kind of validation usually happens in the REST services. Often frameworks have integration with BeanValidation so that you don't need to invoke it yourself.